content/applications/finance/payment_acquirers/alipay.rst

```diff
@@ -5,6 +5,10 @@ Alipay
 `Alipay <https://www.alipay.com/>`_ is an online payments platform established in China by Alibaba
 Group.
 
+.. warning::
+
+    The provider Alipay is deprecated.  It is recommended to use :doc:`asiapay` instead.
```

Comment:
```suggestion
.. warning::
   The provider Alipay is deprecated. It is recommended to use :doc:`asiapay` instead.
```

Reply:
That's actually a correct form of writing according to I-don't-remember-which-university-rulebook :p